### DISCUSSION TITLE
Scanter 2001/4001 land or ship

### DISCUSSION THREAD

**Discussion by Parel803** [PygmalionOfCyprus/cmo-db-requests#3791] [General]
A question:
For sensors on #2615 - Scanter 2001 (Land-based Scanter 4001) and #3323 - Scanter 4100 (Shipborne Scanter 2001)
Does this, in this case means the 2001 is landbased and the 4100 is ships radar? Sorry for my shortcoming in the english language.
Noticed that both radar are connected to Ships in the DB and not to a land facility.
best regards GJ

**Comment by FrangibleCover** [comment]
Yes, that is what the notes indicate. Of course, the notes in the database are not always right!

**Comment by Parel803** [comment]
Thank you
